src/app/api/event/route.js

```diff
@@ -1,6 +1,80 @@
 import { prisma } from '@/lib/db';
 
-export async function GET() {
-  const events = await prisma.event.findMany({ include: { tags: true } });
-  return Response.json(events);
+export async function GET(req) {
+  const today = new Date();
+  const currentYear = today.getFullYear();
+
+  try {
+    const url = new URL(req.url);
+    const take = url.searchParams.get('take');
+    const lastCursor = url.searchParams.get('lastCursor');
+    const queryMonth = url.searchParams.get('month');
+
+    const filteredQueryMonth = parseInt(queryMonth < 10 ? `0${queryMonth}` : queryMonth, 10);
+    const startOfNextMonth = new Date(currentYear, filteredQueryMonth - 1, 1);
+    const endOfMonth = new Date(currentYear, filteredQueryMonth, 0);
+    const endOfNextMonth = new Date(currentYear, filteredQueryMonth, 0);
+
+    const result = await prisma.event.findMany({
+      include: { tags: true, additionalLink: true },
+      where: {
+        isActive: true,
+        eventDate: {
+          gte: filteredQueryMonth === today.getMonth() + 1 ? today : startOfNextMonth,
+          lte: filteredQueryMonth === today.getMonth() + 1 ? endOfMonth : endOfNextMonth,
+        },
+      },
+      take: take ? parseInt(take, 10) : 6,
```

Comment:
@Yivgha default radix

Reply:
we have eslint rule, which demands having radix